### DISCUSSION TITLE
Upd to new minecraft versions.

### DISCUSSION THREAD

**Discussion by Mimiclouse** [fxmorin/carpet-fixes#135] [Q&A] [answered]
When will carpet-fixes be updated to new versions of minecraft 1.19.4 and 1.20?
Will or not?
I wish there was a transfer to new versions as carpet.

**Comment by FxMorin** [comment, marked as answer]
I plan on updating it soon, maybe in 1 week

**Reply by Mimiclouse** [reply]
Thank you very much for response!
